docs/components/user-interface/label.md

```diff
@@ -0,0 +1,42 @@
+# Label
+
+## Overview
+A label is used to show a character.
+
+
+
+<br><br>
+![Preview of button](../../assets/images/component-previews/label.png){: class="preview-img" }

```

Comment:
It has to be Preview of button

Reply:
"../../assets/images/component-previews/label.png" should be "/assets/images/component-previews/label.png"